[DOCS-11412] Document subtest support for Synthetic Multistep API tests (#30284)
DataDog/documentation: content/en/synthetics/multistep.md

{{< img src="synthetics/api_tests/use_variable.mp4" alt="Using Variables in Multistep API tests" video="true" width="90%" >}}

### Subtests

You can run existing API tests within Multistep API tests to reuse existing workflows up to two levels of nesting.

To use an existing API test as a subtest, click **Subtest** and select an API test from the dropdown menu under the **From Existing Test** tab.

To convert steps from your current API test into a subtest, click on the **Extract From Steps** tab, select the recorded steps you want to extract, and click **Convert to Subtest**. 

{{< img src="synthetics/multistep_tests/subtest.png" alt="Add a subtest in a Multistep API test" width="60%" >}}

> In order to override variables from subtests in Multistep API tests, ensure the variables created at the Multistep API test level have the same names as the variables present in the subtest. A variable always uses the value that was first assigned to it. 

Reviewer: Slightly confused by this paragraph. Here's a suggested revision for clarity (am I capturing intended meaning?)

```suggestion
To override a subtest variable in a Multistep API test, define it in the parent test using the same name. A variable always uses the first value assigned to it.
```

> If it does not make sense for you to run your subtest independently, you can pause it. The test continues to be called as part of your Multistep API test, and is not executed individually.

Reviewer: ```suggestion
If you don't need to run a subtest independently, you can pause it. It still runs as part of the Multistep API test but is not executed on its own.
```

> If it does not make sense for you to run your subtest independently, you can pause it. The test continues to be called as part of your Multistep API test, and is not executed individually.

Reviewer 2: ```suggestion
If it does not make sense for you to run your subtest independently, you can pause it. The subtest continues to be run as part of your Multistep API test, but is not executed independently.
```
